packages/patternfly-4/src/components/content/icons.js

```diff
@@ -58,6 +59,20 @@ class Icons extends React.Component {
           return <IconCard key={id} id={id} icon={Icon} name={name} />;
         })}
         </Gallery>
+        <EmptyState variant={EmptyStateVariant.full}>
+          { filteredIcons.length == 0 && 
```

Comment:
Can you bring the check up a level? so `{ filteredIcons.length == 0 && <EmptyState ...>`